pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1593,10 +1595,12 @@ func (c *VolumeBindingChecker) predicate(pod *v1.Pod, meta algorithm.PredicateMe
 		return false, nil, fmt.Errorf("node not found")
 	}
 
+	start := time.Now()
```

Comment:
Instead of adding the start/end timers in the scheduler file, can you put them inside the volume_scheduling.go file?

Reply:
Do you mean we should move the metric to the scheduler_binder.go?

Reply:
Yes that's what I meant, sorry.